openwisp_controller/config/who_is/test_who_is.py

```diff
@@ -81,6 +76,23 @@ def test_who_is_enabled(self):
                 app_settings.WHO_IS_ENABLED,
             )
 
+
+class TestWhoIsTransaction(CreateWhoIsMixin, TransactionTestCase):
+    _WHO_IS_GEOIP_CLIENT = (
+        "openwisp_controller.config.who_is.service.geoip2_webservice.Client"
+    )
+    _WHO_IS_TASKS_INFO_LOGGER = "openwisp_controller.config.who_is.service.logger.info"
+    _WHO_IS_TASKS_WARN_LOGGER = (
+        "openwisp_controller.config.who_is.service.logger.warning"
+    )
+    _WHO_IS_TASKS_ERR_LOGGER = "openwisp_controller.config.who_is.service.logger.error"
+
+    def setUp(self):
+        self.admin = self._get_admin()
+        # connect the signals related to who_is
+        connect_who_is_handlers()
```

Comment:
I guess this could be done in the setUpClass only once per test class instead of once per test?